tests/CordovaLibTests/CordovaLibTests.xcodeproj/project.pbxproj

```diff
@@ -551,7 +563,7 @@
 				PUBLIC_HEADERS_FOLDER_PATH = include/Cordova;
 				SDKROOT = iphoneos;
 				SKIP_INSTALL = YES;
-				SWIFT_OBJC_BRIDGING_HEADER = "__PROJECT_NAME__/Bridging-Header.h";
+				SWIFT_OBJC_BRIDGING_HEADER = "CordovaLibApp/Bridging-Header.h";
```

Comment:
Can you confirm if this is expected?

Reply:
yes, in the test project, the folder name is `CordovaLibApp`. Previously this didn't cause problems because we had no Swift code in the tests, but now it needs to be accurate.